NineAnimator/Models/Anime Source/animeunity.it/AnimeUnity+Featured.swift

```diff
@@ -62,9 +99,10 @@ extension NASourceAnimeUnity {
                 )
             }
             return BasicFeaturedContainer(
-                featured: [],
-                latest: recentAnimeLinks
+                featured: Array(FeaturedAnimeLinks.prefix(10)),
```

Comment:
```suggestion
                featured: Array(featuredAnimeLinks.prefix(10)),
```

Btw, why limit the number of links to 10?